go.mod

```diff
@@ -38,7 +38,7 @@ require (
 	github.com/urfave/negroni v1.0.0
 	github.com/vrischmann/envconfig v1.2.0
 	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8 // indirect
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200401174654-e694b7bb0875
+	go.etcd.io/etcd v0.0.0-20200824191128-ae9734ed278b
```

Comment:
according to [this message](https://github.com/etcd-io/etcd/issues/12258#issuecomment-689503748), you should also bump `google.golang.org/grpc` to `google.golang.org/grpc v1.29.0` (line 56)